fs/mkdirp.ts

```diff
@@ -0,0 +1 @@
+export * from "./mkdirp/mod.ts";
```

Comment:
I don't think you need this file.

`mv mkdirp/mkdirp.ts fs/mkdirp.ts`

(instead of mod.ts) makes for fewer files/directories.

Reply:
Hmm interesting point. I think @ry's idea was that these would be incorporated in the same module (since they are quite small... in this case 25 LOC).

Reply:
Can the readme be moved into a header comment of the file? I’d also prefer as hayd suggested.

Reply:
Should I do the same thing on the `path`?